src/ovp_pipeline/commands/score_domain.py

```
@@ -0,0 +1,278 @@
+        except (yaml.YAMLError, OSError):
+            existing = {}
+    else:
+        existing = {}
+
+    domains = existing.get("domains") or {}
+    domains[host] = {
+        "authority": verdict["authority"],
+        "bucket": verdict["bucket"],
+        "rationale": verdict.get("rationale", ""),
+        "source": source,
+        "added_at": date.today().isoformat(),
```

Ruff (0.15.12)

[warning] `datetime.date.today()` used

(DTZ011)